src/tests/ftest/util/general_utils.py

```diff
@@ -134,6 +134,144 @@ def _calculate_metrics(cls, data):
         return max_time, min_time, avg_time
 
 
+class RemoteCommandResult():
+    """Stores the command result from a Task object."""
+
+    class ResultData():
+        # pylint: disable=too-few-public-methods
+        """Command result data for the set of hosts."""
+
+        def __init__(self, command, returncode, hosts, stdout, timeout):
+            """Initialize a ResultData object.
+
+            Args:
+                command (str): the executed command
+                returncode (int): the return code of the executed command
+                hosts (NodeSet): the host(s) on which the executed command yielded this result
+                stdout (list): the result of the executed command split by newlines
```

Comment:
What about stderr?

Reply:
The way we execute the command (`task.run()` where `stderr=False` is the default) the stdout and stderr are combined.